### ISSUE TITLE
Response data persistence

### ISSUE THREAD

**Issue by Art1k** [SmartBear/ready-api-testserver-client#12]
The problem is the following: if you have API method that returns some SessionID after login,and then it should be passed as header or as part of the request,as authorisation parameter. 
In this case I'd like to login once (recipe A) and execute all other operations (recipe B,C,etc) using SessionID from recipe A.
For now there's no option to persist data between recipes.

I'm using version 1.0.2

**Comment by olensmar** [comment]
hm... I don't think the TestServer currently maintains any state that could be shared between recipe executions - perhaps we could introduce a "bucket" concept to/from which test recipes can read property values - or alternatively you would have to submit all depending test-recipes within one transaction (i.e. like you would do with a soapui-project containing multiple test-cases) - within they could define and refer to input/output properties from each-other!?

@spindelmanne thoughts on this?

**Comment by Art1k** [comment]
Yes, it would be nice to have sort of bucket that can keep data between recipes like it's implemented now on Project level, it's just that it seems not so robust to write xml project for each case you need shared value
